tensorly/backend/__init__.py

```diff
@@ -224,6 +224,7 @@ def inner(*args, **kwargs):
 finfo = dispatch(Backend.finfo)
 index = Backend.index
 index_update = dispatch(Backend.index_update)
+log2 = dispatch(Backend.log2)
```

Comment:
We need to add `log2` to Cupy, JAX, and TensorFlow too.

Reply:
Yep, indeed. Working on that for sure.